index.html

```diff
@@ -5,63 +5,77 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
   <title>Karel's Portfolio Home Page</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">  <link rel="stylesheet" href="style.css">
-
 </head>
 
 <body>
   <section class="main-header">
 
-      <nav class="navbar-mobile">
-          <a class="mobile-logo">My Logo</a>
-          
-          <a class="toolbar-button" href=#>
-            <img src="images/header/toolbar/Union-button.png" alt="toolbar menu">
+        <nav class="navbar-mobile" id="navbar-mobile">
+            <a class="mobile-logo" id="mobile-logo">Karel van Oordt</a>
+            
+            <a class="toolbar-button" href=# id="hamburger">
+              <img src="images/header/toolbar/Union-button.png" alt="toolbar menu">
+            </a>
+        </nav>
+
+      
+        <nav class="navbar-menu display-none">
+          <a class="exit-link" href="#" id="exit-link">
+            X
           </a>
-      </nav>
 
-      <nav class="navbar-desktop">
+          <ul class="menu-items">
+            <li ><a class="menu-option" href="#portfolio">Portfolio</a></li> 
+            <li ><a class="menu-option" href="#about">About</a></li> 
+            <li ><a class="menu-option" href="#contact-container">Contact</a></li> 
+          </ul>
+        </nav>
```

Comment:
- [ ] Kindly ensure that the nav display looks exactly like the given template. Adjust the spacing between nav items to match the given design and also ensure that they have the same background color.

| Original | Yours |
| -------- | ----- |
| <img src="https://user-images.githubusercontent.com/33323077/135995925-09fb42dd-f3c8-4a05-8cca-6f585edf0c1f.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/135996001-cd3bdcb7-f169-41c4-90e6-5c8aafbbdc57.png" height="250px"/>

Reply:
Thanks for the heads up, Mike! 👍